bin/generate-podspecs.sh

```diff
@@ -7,23 +7,48 @@ set -e
 cd "$( dirname $0 )"
 cd ..
 
-# Check for cocoapods
+# Check for cocoapods & jq
 command -v pod > /dev/null || ( echo Cocoapods is required to generate podspecs; exit 1 )
+command -v jq > /dev/null || ( echo jq is required to generate podspecs; exit 1 )
 
-PODSPECS=$(cat <<EOP
-node_modules/react-native/React.podspec
-node_modules/react-native/ReactCommon/yoga/yoga.podspec
-node_modules/react-native/third-party-podspecs/Folly.podspec
-node_modules/react-native/third-party-podspecs/DoubleConversion.podspec
-node_modules/react-native/third-party-podspecs/glog.podspec
-EOP
-)
+WD=$(pwd)
+DEST="${WD}/react-native-gutenberg-bridge/third-party-podspecs"
 
-DEST="react-native-gutenberg-bridge/third-party-podspecs"
+# Generate the external (non-RN podspecs)
+EXTERNAL_PODSPECS=$(find "node_modules/react-native/third-party-podspecs" \
+                         "node_modules/react-native-svg" \
+                         "node_modules/react-native-keyboard-aware-scroll-view" \
+                         "node_modules/react-native-recyclerview-list" \
+                         "node_modules/react-native-safe-area" -type f -name "*.podspec" -print)
 
-for podspec in $PODSPECS
+for podspec in $EXTERNAL_PODSPECS
 do
-    pod=`basename $podspec .podspec`
+    pod=$(basename "$podspec" .podspec)
+
     echo "Generating podspec for $pod"
-    INSTALL_YOGA_WITHOUT_PATH_OPTION=1 pod ipc spec $podspec > "$DEST/$pod.podspec.json"
-done
\ No newline at end of file
+    pod ipc spec $podspec > "$DEST/$pod.podspec.json"
+done
+
+# Generate the React Native podspecs
+# Change to the React Native directory to get relative paths for the RN podspecs
+cd "node_modules/react-native"
+
+RN_PODSPECS=$(find * -type f -name "*.podspec" -not -path "third-party-podspecs/*" -print)
```

Comment:
Now using `find` to generate all the podspecs in `react-native`. This means we have a lot more podspecs in the repo (hence the big diff).